principles/fp-019-term-stability.md

```diff
@@ -0,0 +1,144 @@
+
+Recommendations and Requirements
+-------
+
+If changing a term definition would change its referents, then instead a new term MUST be created with a new IRI and the new definition. Minor changes to the definition for clarity, grammar, and/or proper punctuation that do not change the referents are permitted. What is considered a 'minor change' will likely need to be considered on a case-by-case basis; it is left to the ontology developers to decide. However, any feedback from users MUST be taken into account.
+
+Conditions under which a term must be deprecated according to this principle, or for which term deprecation should be considered, include:
+
+1) The old textual definition misses its intended target. This includes cases where the term refers to non-existent referents (as might happen, for example, when new research reveals that the referent does not exist in reality).
+1) The original term definition is considered sufficiently “damaged” (too vague, too restrictive, too misused or too misunderstood).
+
+In all cases the developers SHOULD provide guidance on how to handle deprecated terms (either by exact replacement or by considering other terms), and be mindful of the potential costs to users of the ontology who might use the existing term. As well, developers SHOULD pre-announce term obsoletions. See [Principle 13](http://obofoundry.org/principles/fp-013-notification.html) for guidance on such announcements.
+
+Implementation
+-------
+
+Detailed procedures for obsoleting a term are described on the OBO Academy page [Obsoleting an Existing Ontology Term](https://oboacademy.github.io/obook/howto/obsolete-term/). 
+
+<i><b>To obsolete a term, the ontology developer</b></i> MUST:
+1) Mark the term as obsolete
+  - OWL format: Add an "owl:deprecated" annotation property with value of "true^xsd:boolean"
+  - OBO format: Add an "is_obsolete: true" tag
+2) Prepend the string "obsolete " (including the space) to the term label
+  - NOTE: To be consistent with [Principle 12](https://obofoundry.org/principles/fp-012-naming-conventions.html) "Naming Conventions", the syntax/format MUST be precisely as given above (that is, the exact string as shown, lowercase and space included, with no other punctuation before or after). Thus, the following are disallowed: "Obsolete {label}", "obsolete_{label}", "OBSOLETE {label}" (and variations thereof).
+3) Remove all existing logical axioms from the term
+4) Remove or replace all usages of the term elsewhere in the ontology
+
+It is not necessary (and not advisable) to delete the textual definition.
+
+<i><b>To obsolete a term, the ontology developer</b></i> SHOULD:
+1) Indicate any exact term replacement:
+   -  OWL: Use the "term replaced by" annotation property from OMO ([IAO:0100001](http://purl.obolibrary.org/obo/IAO_0100001)) with the value set to the IRI of the relevant term
```

Comment:
I recommend backticks over double quotes in markdown when referring to controlled terms, curies, etc

Reply:
Hmm, I think we've not been terribly consistent over all the principles. I'll put this on our to-do list (that is, I'll make an issue). https://github.com/OBOFoundry/OBOFoundry.github.io/issues/2669